src/components/listing/OpeningListing.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './EmployeeListing.css';
+import OpeningListItem from '../list-item/OpeningListItem';
+import { useGetOpeningListQuery } from '../../services/openingApi';
+
+type OpeningListingPropsType = {
+  labels: string[];
+};
+
+const OpeningListing: React.FC<OpeningListingPropsType> = (props) => {
+  const { data: openingData, isSuccess } = useGetOpeningListQuery();
+
+  const labels = props.labels.map((label) => (
+    <td className='listing-label' key={label}>
+      <label>{label}</label>
+    </td>
+  ));
+
+  let openings = [];
```

Comment:
I think it's better to keep openings as a state and then set it during the useEffect of isSuccess. I did the same way as here for EmployeeListing and it works, but I think using useEffect would be a safer practice.

Reply:
I've made this change for EmployeeListing.tsx in referral-form branch.